mmv1/templates/terraform/examples/kmsConfig_create.tf.erb

```diff
@@ -0,0 +1,6 @@
+resource "google_netapp_kmsconfig" "<%= ctx[:primary_resource_id] %>" {
+  kms_name = "<%= ctx[:vars]['kms_name'] %>"
+  description="this is a test description"
+  crypto_key_name="projects/cxo-automation-gcp/locations/us-central1/keyRings/kmsRing2/cryptoKeys/kmsCrypto111"
+  location="usthis is a test description-central1"
```

Comment:
>        Error: Error creating kmsconfig: googleapi: Error 403: Permission denied on 'locations/usthis is a test description-central1' (or it may not exist).
        
          with google_netapp_kmsconfig.kmsConfig,
          on terraform_plugin_test.tf line 2, in resource "google_netapp_kmsconfig" "kmsConfig":
           2: resource "google_netapp_kmsconfig" "kmsConfig" {

appears to be a problem here! also could you update this sample to be using a cryptokey terraform resource for the  crypto_key_name